components/tooltip/test/tooltip.visual-diff.js

```diff
@@ -62,4 +62,21 @@ describe('d2l-tooltip', () => {
 
 	});
 
+	[
+		'position-top',
+		'position-bottom',
+		'position-right',
+		'position-left'
+	].forEach((testName) => {
+
+		it(`dark-background-${testName}`, async function() {
+			await page.addStyleTag({ content: 'body { background-color: black }' });
```

Comment:
If these sets of tests were in their own `describe` block, then the style could be added just once in `before(...)`, and it could be cleaned up in `after(...)`. Though Puppeteer is probably smart enough not to add it twice, and it would be a no-op anyway. Not a big deal since there are no other tests that follow this set.

Reply:
The tests have an `afterEach(async() => await page.reload());`, so I still need to apply the styles before each test - but I like moving them into a describe block and doing that styling with a `beforeEach`.

Reply:
Ah cool... 👍 .  I didn't notice the reload. 😄